modules/nodes-adding-bare-metal-host-to-cluster-using-cli.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-cli_{context}"]
+= Adding a bare metal host to the cluster and provisioning a bare metal node using the CLI
+
+You can add bare metal hosts to the cluster and then provision them as bare metal nodes using the CLI.
+
+.Prerequisites
+
+* Install a {op-system} compute machine on bare metal infrastructure for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a bare metal host for use in the cluster by completing the following actions:
+.. Create the following `BareMetalHost` Custom Resource (CR) that defines the `Secret` credentials and the required cluster node namespace, for example `openshift-worker`. Save the YAML in the `openshift-worker-namespace.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-bmc-secret
+type: Opaque
+data:
+  username: <secret-username>
+  password: <secret-password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker
+spec:
+  online: true
+  bmc:
+    address: <BMC address>
+    credentialsName: openshift-worker-bmc-secret
+    disableCertificateVerification: True
+    username: <username>
+    password: <password>
+  bootMACAddress: <host boot MAC address>
+  hardwareProfile: unknown
+  providerSpec:
+    value:
+      hostSelector: <1>
+        matchLabels:
+          key: <host label> <2>
+----
+<1> Optional. Used to restrict automatic `Machine` creation from `MachineSet` objects. When specified, `Machine` objects are created from `MachineSet` objects only when the same label exists on an unused `BareMetalHost` object.
```

Comment:
@mhrivnak Added detail about hostSelector as a supplementary note, and updated the  sample `BareMetalHost` yaml to include this field.